### ISSUE TITLE
[med] Chrome + Jaws: Role is not defined for Send button.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8480]
## Action Performed:
Break down in numbered steps

## Expected Result:
Role = "Button" should be defined for send control, so that user can easily identify the function of the control.

## Actual Result:
Role is not defined for Send button. Screen reader is only reading "Send".

##Note - On MAC screen reader is reading "Send group".

## Workaround:
Yes

## Area issue was found in:
Send Window

## Failed WCAG checkpoints
4.1.2

## User impact:
Screen reader user might be confused without knowing the role of the interactive element, so the user is not able to take any action over the control.

## Suggested resolution:
Add the role="button" for the send control or implement the element as a native button, so that the role can be announced by screen reader.                                                                                                                             Refer to:                                                                                                          https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role    
## Platform:
- Web
- iOS
- Desktop App
- Mobile Web

**Version Number:**  v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/161669266-393e2add-1c7f-466c-9753-f6de7200ac86.mp4
